ccd-definition/AuthorisationCaseField/CareSupervision/magistrate.json

```diff
@@ -1014,13 +1014,6 @@
     "UserRole": "caseworker-publiclaw-magistrate",
     "CRUD": "CRU"
   },
-  {
```

Comment:
As we discussed this role needs to have other fields?

Reply:
Ah sorry, yes added back in.